plugins/algoltek-usb/algoltek-usb.quirk

```diff
@@ -0,0 +1,3 @@
+# AG9421 
```

Comment:
leading whitespace issue will be fixed by pre-commit

Reply:
Updated!